### PR TITLE
Fix: advanced tools issues

### LINKED ISSUE #715: "Generate random temporary directory" setting is redirecting to /wp-admin/options.php
Currently, when you click on the "Generate random temporary directory" setting, it is redirecting to `/wp-admin/options.php`.

### PR DESCRIPTION
**Pull request by alexmigf**
closes #715

### CODE REVIEW

File: includes/views/advanced-tools.php [hunk 1]
```diff
@@ -8,9 +8,11 @@
 			<h4><?php _e( 'Generate random temporary directory', 'woocommerce-pdf-invoices-packing-slips' ); ?></h4>
 			<p><?php _e( 'For security reasons, it is preferable to use a random name for the temporary directory.', 'woocommerce-pdf-invoices-packing-slips' ); ?></p>
 			<form method="post">
-				<?php wp_nonce_field( 'wpo_wcpdf_debug_tools_action', 'security' ); ?>
-				<input type="hidden" name="wpo_wcpdf_debug_tools_action" value="generate_random_string">
-				<input type="submit" name="submit" id="submit" class="button" value="<?php esc_attr_e( 'Generate temporary directory', 'woocommerce-pdf-invoices-packing-slips' ); ?>">
+				<input type="hidden" name="debug_tool" value="generate_random_string">
+				<a href="" class="button button-secondary submit"><?php _e( 'Generate temporary directory', 'woocommerce-pdf-invoices-packing-slips' ); ?></a>
```
Comment (MohamadNateqi) [inline includes/views/advanced-tools.php:12]: Maybe using an `<input type="submit">` was better for compatibility and accessibility?

**Review by MohamadNateqi — COMMENTED** [review]
It would be good to normalize the path:
image

**Review by YordanSoares — APPROVED** [review]
LGTM!

**Review by dwalkerpriv — APPROVED** [review]
Nice 👍

**Review by MohamadNateqi — COMMENTED** [review]
The 'Clear Professional extension license cache' option added by the Pro plugin no longer works. We either need to update that plugin accordingly or update this PR to be compatible.

**Comment by alexmigf** [comment]
I know, it's fixed here: https://github.com/wpovernight/woocommerce-pdf-ips-pro/pull/365

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Review by YordanSoares — APPROVED** [review]
Tested all the Debug tools, just in case, and everything seems to be working correctly.

**Review by dpeyou — APPROVED** [review]
Success!

generate temporary directory

**Pull request merged**